src/plugins/fs_routes/mod.ts

```diff
@@ -221,12 +221,10 @@ export async function fsRoutes<State>(
 
       // _app template
       if (skipApp && mod.path === "/_app") {
-        hasApp = false;
         continue;
       } else if (!skipApp && mod.config?.skipAppWrapper) {
         skipApp = true;
         if (hasApp) {
-          hasApp = false;
           // _app component is always first
           components.shift();
```

Comment:
I don't know if it's correct just to remove these lines. It looks wrong to me to set `hasApp` to `false` if we're looping over every route, since a latter route may have the app wrapper disabled always.

This change makes the test I added pass anyway...